mw-webapp/src/styles/_variables.scss

```diff
@@ -8,6 +13,13 @@ $transparent-black: rgba(0, 0, 0, 90%);
 $borderBlack: rgb(0, 0, 0);
 $blackBackground: rgb(0, 0, 0);
 $whiteText: rgb(255, 255, 255);
```

Comment:
Don't use specific things in global scope, think about purpose and making

Reply:
Done